docs/articles/posts/leadership-playbook-change.md

```diff
@@ -0,0 +1,93 @@
+---
+draft: false
+date: 2024-06-18
+authors:
+  - cameronpresley
+description: >
+  Leadership Playbook - Leading Through Change
+categories:
+  - Leadership
+hide:
+  - toc
+---
+
+# Leadership Playbook - Leading Through Change
+
+There are few fundamental truths in life, one of which is that the only consistent thing is change. Whether that's through a reorganization, someone leaving the team, or the start of a new initiative, we know that what happens today is different from yesterday, and as a leader, the team is looking to you to figure out how to navigate these changes.
+
+In this post, I'll share some tips and tricks for leading the team through change.
+
+## Explain the Reasoning
+
+A common mistake I see leaders make is introducing a change without discussing _why_ the change is happening. Let's look at a hypothetical situation where you're introducing the team to pull request templates.
+
+> Hey team! Starting next sprint, we're going to starting using this new PR template. You can find it ....
+
+On the one hand, the message is clear on the what's changing (using new PR template) and what's changing. However, it completely missed the point, why are we making this change? When we don't include the why, we catch people off-guard because they may not immediately understand the problem that the change is supposed to solve.
```

Comment:
maybe duplicative copy here with the latter 'and what's changing.'

_the message is clear on the **what's changing** (using new PR template) **and what's changing._**